.github/workflows/package-release.yml

```diff
@@ -77,142 +80,24 @@ jobs:
       - name: "Copy signature file"
         run: cp ${RELEASE_ASSETS}/${{ env.PACKAGE_FILE }}.sig .
 
-      - name: "Upload artifacts"
-        uses: actions/upload-artifact@v4
-        with:
-          name: ${{ env.PACKAGE_FILE }}
-          path: |
-            ${{ env.PACKAGE_FILE }}
-            ${{ env.PACKAGE_FILE }}.sig
-          retention-days: 3
-
       - name: "Upload release artifacts"
-        run: gh release upload ${{ github.ref_name }} ${{ env.PACKAGE_FILE }} ${{ env.PACKAGE_FILE }}.sig
-        continue-on-error: true
```

Comment:
It was. I put it there initially because any tag build would trigger the package workflow, regardless of whether there was a GitHub release to go with it. With the change, the package workflow is only triggered from the release workflow, so if the upload fails we want to be alerted instead of the workflow run passing as if nothing went wrong.